app/boards/shields/helix/helix.dtsi

```diff
@@ -4,8 +4,6 @@
  * SPDX-License-Identifier: MIT
  */
 
-#include <dt-bindings/zmk/matrix-transform.h>
-
```

Comment:
This shouldn't have been removed. It should have been *updated* to matrix_transform. Currently this shield will fail to build.

Reply:
Thank you, I added the corrected text on line 7 and will add it in my next push.